docs/docs-content/troubleshooting/enterprise-install.md

```diff
@@ -10,6 +10,108 @@ tags: ["troubleshooting", "self-hosted", "palette", "vertex"]
+
+<TabItem value="node1" label="Example output from node 1">
+
+````shell hideClipboard
+
+```shell hideClipboard title="Example output from node 1"
+               Local time: Fri 2025-07-11 09:41:42 UTC
+           Universal time: Fri 2025-07-11 09:41:42 UTC
+                 RTC time: Fri 2025-07-11 09:41:42
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: no
+              NTP service: active
+          RTC in local TZ: no
+````
+
+</TabItem>
+
+<TabItem value="node2" label="Example output from node 2">
+
+```shell hideClipboard
+               Local time: Fri 2025-07-11 09:41:45 UTC
+           Universal time: Fri 2025-07-11 09:41:45 UTC
+                 RTC time: Fri 2025-07-11 09:41:45
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: no
+              NTP service: active
+          RTC in local TZ: no
+```
+
+</TabItem>
+
+<TabItem value="node3" label="Example output from node 3">
+
+```shell hideClipboard
+               Local time: Fri 2025-07-11 09:41:49 UTC
+           Universal time: Fri 2025-07-11 09:41:49 UTC
+                 RTC time: Fri 2025-07-11 09:41:49
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: no
+              NTP service: active
+          RTC in local TZ: no
+```
+
+</TabItem>
+
+</Tabs>
+
+<!-- You may also notice errors in the auth service logs similar to the following.
+
+```shell hideClipboard
+auth-5f95c77cb-xslmk Jul  7 12:35:30.306 ERROR [err_logger.go:47/root.LogError] [reqId:r1751891730295getb911939434c9] [Unable to read the 'Authorization' cookie dat
+a] [arg1=[null] code=CookieReadError mtd=GET ref=xyz uri=/v1/auth/cookie/refresh ]
+auth-5f95c77cb-xslmk Jul  7 17:02:22.548 INFO [auth_broker.go:46/service.(*AuthBrokerService).AuthBroker] [reqId:r1751907742548pose5396e802750] [Client service 'all
+y' got authorized] [orgId= sessionId=ally5a03 ]
+auth-5f95c77cb-xslmk Jul  7 17:22:12.673 ERROR [hubble_token.go:426/hucontext.getClaimsFromToken] [Unable to parse the token 'abcd...1234' due to Token used before
+issued]
+auth-5f95c77cb-xslmk Jul  7 17:22:16.004 INFO [auth_login.go:450/internal.(*AuthLoginService).CreateUserToken] [reqId:r1751908935214posa171c4b915ad] [Auth success:
+12345abc67890def12345abc,johndoe/johndoe@company.com - Going for MFA false] [orgId=johndoe tenantUid=12345abc67890def12345abc ]
+auth-5f95c77cb-xslmk Jul  7 17:22:46.378 ERROR [hubble_token.go:426/hucontext.getClaimsFromToken] [Unable to parse the token 'abcd...1234' due to Token used before
+issued]
+auth-5f95c77cb-xslmk Jul  7 17:22:46.378 ERROR [auth_service.go:282/service.(*AuthService).Logout] [provided token 'xxxxx' is not valid Token used before issued]
+auth-5f95c77cb-xslmk Jul  7 17:22:57.473 INFO [auth_login.go:450/internal.(*AuthLoginService).CreateUserToken] [reqId:r1751908976769pos6533ca18ccb3] [Auth success:
+12345abc67890def12345abc,johndoe/johndoe@company.com - Going for MFA false] [orgId=johndoe tenantUid=12345abc67890def12345abc ]
+``` -->
```

Comment:
Hidden this for now as I'm awaiting the exact log file where QA provided this example.

It would be a nice enhancement, although it's not entirely needed.

Reply:
I got the details from QA and have now included this part.